doc/classes/TranslationServer.xml

```diff
@@ -65,10 +73,11 @@
 				Returns a readable language name for the [param language] code.
 			</description>
 		</method>
-		<method name="get_loaded_locales" qualifiers="const">
+		<method name="get_loaded_locales" qualifiers="const" deprecated="">
 			<return type="PackedStringArray" />
 			<description>
 				Returns an array of all loaded locales of the project.
+				[b]Note:[/b] This method is deprecated. Use [method has_translations] to check if translations are loaded for a certain locale. Use [method get_translations] or [method find_translations] to get a list of loaded translations. Godot supports multiple translation objects per locale.
```

Comment:
You should move this to the deprecated note. I'm not sure why this line has been separated.